### PR TITLE
hoc2018: robotics announcements on hourofcode.com for soon-hoc

### PR DESCRIPTION
**Pull request by breville**
Fully responsive.  Screenshots are widest browser to narrowest.

screenshot 2018-10-10 17 11 30

screenshot 2018-10-10 17 11 49

screenshot 2018-10-10 17 12 04

screenshot 2018-10-10 17 12 28

screenshot 2018-10-10 17 12 39

### CODE REVIEW

File: pegasus/sites.v3/hourofcode.com/public/css/front-page.css [hunk 1]
```diff
@@ -48,8 +48,20 @@ a.category-link {
...
+.robotics-providers {
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap
+}
+
+.robotics-provider {
+  width: 182px;
+  height: 210px;
+  xfloat: left;
```
Comment (Erin007) [inline pegasus/sites.v3/hourofcode.com/public/css/front-page.css]: typo or is "xfloat" something I don't know about?

Comment (breville) [reply pegasus/sites.v3/hourofcode.com/public/css/front-page.css]: Oops, good catch.  It was a cheap way to "comment out" the line.

**Comment by tanyaparker** [comment]
My desktop width looks like this.

image

When I pictured the responsiveness of this I thought the robot kits would start as 2 columns, always half of the size of the announcement column. And as the width decreases, each robot box would decrease in width until it hit its min width (the current size you have now). And once it hits this min width it would collapse to 1 column. I guess I'm kind of confused how it switches to a 3 column. Can the robot kits be flexible in size to take up the awkward spacing?

**Comment by breville** [comment]
@tanyaparker Good catch.  Bootstrap supported one wider layout than I was using.  I've changed the layout to prefer 2 columns at all times unless the items get smaller than a minimum width, and made the items stretch.  Unfortunately, until flex gets a "gap" property, I don't know a good way to make the right side flush as the left side is, without having the items sometimes touching each other.  Screenshots in description updated.

**Pull request merged**